examples/webpack.config.js

```diff
@@ -42,26 +43,28 @@ module.exports = async (env) => {
 
     // see also production proxy at /examples/server/src/endpoints/proxy.js
     const proxy = {
-        '/gdc': {
-            target: backendUrl,
-            secure: false,
-            cookieDomainRewrite: '',
-            onProxyReq: (proxyReq) => {
-                console.log('proxy', '/gdc', proxyReq.path); // eslint-disable-line no-console
-                if (proxyReq.method === 'DELETE' && !proxyReq.getHeader('content-length')) {
-                    // Only set content-length to zero if not already specified
-                    proxyReq.setHeader('content-length', '0');
-                }
-
-                proxyReq.setHeader('host', backendUrl.split('/')[2]); // White labeled resources are based on host header
-                proxyReq.setHeader('referer', backendUrl);
-                proxyReq.setHeader('origin', null);
+        "/gdc": {
+            "changeOrigin": true,
```

Comment:
Formatting nitpick: the quotes around the key names are not necessary

Reply:
Odd, that prettier did not catch this. I will fix that